CONTRIBUTING.md

```diff
@@ -175,6 +171,12 @@ git checkout -b myfeature
 
 You can now edit the code on the `myfeature` branch.
 
+#### Download Dependencies
```

Comment:
Is this needed? If we run the `make server` as in `Run stand-alone mode`, go mod can automatically download it for us.

Reply:
there is no need to do this, I will remove it.